.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+version: 2.1
+
+orbs:
+  codacy: codacy/base@1.2.1
+
+references:
+  default_golang_image: &default_golang_image
+    docker:
+      - image: golang:1.13.7-alpine3.11
+    working_directory: ~/workdir/
+
+  get_requirements: &get_requirements
+    run:
+      name: Install requirements
+      command: |
+        apk add git alpine-sdk
+        go get github.com/pborman/getopt
+        go get github.com/stretchr/testify/assert
+  attach_to_workspace: &attach_to_workspace
+    attach_workspace:
+      at: ~/
```

Comment:
If you update the codacy orb to 2.x remember to change it to:
```yml
    attach_workspace:
      at: ~/workdir
```
[Source](https://github.com/codacy/codacy-orbs/blob/master/CHANGELOG.md#200)